intersected_cell.cpp

```diff
@@ -1,3 +1,23 @@
+/******************************************************************************
+   Copyright (C) 2015-2016 Einar J.M. Baumann <einar.baumann@gmail.com>
+???? Copyright (C) 2015-2016 Hilmar M. Magnusson <hilmarmag@gmail.com>
```

Comment:
Well... I created and wrote the file, so I guess not, but it's not that important.